buildkite/src/Command/Base.dhall

```diff
@@ -95,6 +95,7 @@ let Config =
       , artifact_paths : List SelectFiles.Type
       , env : List TaggedKey.Type
       , label : Text
+      , timeout_in_minutes : Text
```

Comment:
If you add this here, then you should also add a default for it just below this (and set to 120 I imagine), otherwise this will require all consumers of this to provide the timeout manually.

you could also delete this line and only hardcode it to Some 120 below as you’ve done.

If you want this to be configurable, then down below you’d instead say Some c.timeout_in_minutes